db-commons-test/src/test/scala/org/bitcoins/db/DbManagementTest.scala

```diff
@@ -48,34 +49,69 @@ class DbManagementTest extends BitcoinSAsyncTest with EmbeddedPg {
                                         dbConfig(ProjectType.Chain))
     val chainDbManagement = createChainDbManagement(chainAppConfig)
     val result = chainDbManagement.migrate()
-    val expected = chainAppConfig.driver match {
-      case SQLite     => 5
-      case PostgreSQL => 4
+    chainAppConfig.driver match {
+      case SQLite =>
+        val expected = 5
+        assert(result == expected)
+        val flywayInfo = chainDbManagement.info()
+        assert(flywayInfo.applied().length == expected)
+        assert(flywayInfo.pending().length == 0)
+      case PostgreSQL =>
+        val expected = 4
+        assert(result == expected)
+        val flywayInfo = chainDbManagement.info()
+        //+1 for << Flyway Schema Creation >>
+        assert(flywayInfo.applied().length == expected + 1)
```

Comment:
Maybe a better solution is to add to `DbManagement` an `appiedMigrations` function and on postgres we just call `_.tail`. Otherwise if we want to add handling for if stuff between migrations we'll have to remember that these differ as well.

Reply:
Let's see if this becomes a bigger issue, for now i think i'm going to leave it as is